app/src/main/java/io/homeassistant/companion/android/notifications/MessagingManager.kt

```diff
@@ -1359,8 +1359,9 @@ class MessagingManager @Inject constructor(
         }
 
     private fun Bitmap.getCompressedFrame(): Bitmap? {
-        val newHeight = height / 4
-        val newWidth = width / 4
+        val ratio: Float = (width.toFloat() / height.toFloat())
+        val newHeight = (480 / ratio).toInt()
+        val newWidth = 480
```

Comment:
> Wouldn't this unnecessarily increase the size for videos <480px wide?

Yes, yes it would. I completely overlooked that scenario. Let me add some logic to only shrink if the initial width is less than 480p